src/upload_all_java_tools.sh

```diff
@@ -60,9 +60,6 @@ for java_version in 11; do
         # Non-Windows needs "file:///foo/bar".
         file_url="file://${zip_path}"
     fi
-    bazel test --verbose_failures --test_output=all --nocache_test_results \
-      //src/test/shell/bazel:bazel_java_test_local_java_tools_jdk${java_version} \
-      --define=LOCAL_JAVA_TOOLS_ZIP_URL="${file_url}"
```

Comment:
Instead of removing this test, may be comment it out, in case @meteorcloudy will find a way to fix too long command line on Windows?

Reply:
That's a good idea. You probably want to also enable it while doing the manual release?

Reply:
{quote}
You probably want to also enable it while doing the manual release?
{quote}

Right. This test is passing in sane platforms: Linux and Mac Os X. And shoud be executed there and only skipped on Windows, so may be even:

```bash
# Skip for now, as the test is broken on Windows.
# See https://github.com/bazelbuild/bazel/issues/12244 for details
if not "$is_windows"; then
    bazel test --verbose_failures --test_output=all --nocache_test_results \
          //src/test/shell/bazel:bazel_java_test_local_java_tools_jdk${java_version} \
          --define=LOCAL_JAVA_TOOLS_ZIP_URL="${file_url}"
fi
```

Reply:
Following @davido suggestion, so we don't have commented out code.

I don't have a setup to do manual releases. Also there is a comment in the script advising against doing them.